website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,211 @@ Learn more about Karpenter and how to get started below.
+
+If you have another option for using IAM credentials with workloads (e.g. [kube2iam](https://github.com/jtblin/kube2iam)) your steps will be different.
+
+Set a variable for your cluster name.
+
+```bash
+CLUSTER_NAME=<your cluster name>
+```
+
+Set other variables from your cluster configuration.
+
+{{% script file="./scripts/step04-env.sh" language="bash" %}}
+
+Use that information to create our IAM role, inline policy, and trust relationship.
+
+{{% script file="./scripts/step05-controller-iam.sh" language="bash" %}}
+
+### Add tags to subnets and security groups
+
+Next add tags to your nodegroup subnets so Karpenter will know which subnets to use.
+
+{{% script file="./scripts/step06-tag-subnets.sh" language="bash" %}}
+
+Add tags to the security groups.
+This command only tags the security groups for the first nodegroup in the cluster.
+If you have multiple nodegroups or multiple security groups you will need to decide which one Karpenter should use.
+
+{{% script file="./scripts/step07-tag-security-groups.sh" language="bash" %}}
+
+### Deploy Karpenter
+
+First set the Karpenter release you want to deploy and the IAM instance profile that is used by the existing nodes in your cluster.
+```bash
+export KARPENTER_VERSION={{< param "latest_release_version" >}}
+export IAM_INSTANCE_PROFILE_NAME=<your instance profile name>
+```
+
+We can now generate a full Karpenter deployment yaml from the helm chart.
+
+{{% script file="./scripts/step09-generate-chart.sh" language="bash" %}}
+
+Modify the following lines in the karpenter.yaml file.
+
+### Set node affinity
+
+Edit the karpenter.yaml file and find the karpenter deployment affinity rules.
+Modify the affinity so karpenter will run on one of the existing node group nodes.
+
+The rules should look something like this.
+Modify the value to match your `$NODEGROUP`.
+
+```
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: karpenter.sh/provisioner-name
+                operator: DoesNotExist
+            - matchExpressions:
+              - key: eks.amazonaws.com/nodegroup
+                operator: In
+                values:
+                - ${NODEGROUP}
+```
+
+Now that our deployment is ready we can create the karpenter namespace, create the provisioner CRD, and then deploy the rest of the karpenter resources.
+
+{{% script file="./scripts/step10-deploy.sh" language="bash" %}}
+
+## Create default provisioner
+
+We need to create a default provisioner so Karpenter knows what types of nodes we want for unscheduled workloads.
+You can refer to some of the [example provisioners](https://github.com/aws/karpenter/tree{{< githubRelRef >}}examples/provisioner) for specific needs.
+
+{{% script file="./scripts/step11-create-provisioner.sh" language="bash" %}}
+
+## Set nodeAffinity for critical workloads (optional)
```

Comment:
WDYT about excluding this from guidance? This is more of a personal choice. Personally, I'd be happy to run both of these on karpenter nodes, and I don't see a great reason to recommend against it.

Reply:
^ Agree, I don't think we'll need this just for the getting-started guide, since it's more of a "do you want more availability on your workloads"? Which is more of a Kubernetes good idea rather than something integral to the installation.